tools/pipelines/templates/1ES/build-docker-service.yml

```diff
@@ -161,15 +161,16 @@ extends:
           - name: containerTag
             value: $(containerRegistryUrl)/$(buildContainerName):$(containerTagSuffix)
           - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/main') }}:
-            - name: latestContainerTagSuffix
-              value: latest
```

Comment:
`latestContainerTagSuffix` was only used as part of the next variable declaration, no point in having it as a variable in itself so I inlined it (here and a few lines below).